mesonbuild/ast/printer.py

```diff
@@ -66,9 +66,7 @@ def visit_NumberNode(self, node: mparser.NumberNode) -> None:
         node.lineno = self.curr_line or node.lineno
 
     def escape(self, val: str) -> str:
-        return val.translate(str.maketrans(T.cast(
-            'T.Dict[str, T.Union[str, int]]',
-            {'\'': '\\\'', '\\': '\\\\'})))
+        return val.replace('\\', '\\\\').replace("'", "\'")
```

Comment:
I'm really shocked here, but this is about 10x faster than the `str.translate` approach. I expected the opposite.